src/content/smart-contracts/index.md

```diff
@@ -5,21 +5,15 @@ lang: en
 sidebar: true
 ---
 
-# Introduction to smart contracts {#introduction-to-smart-contracts}
+Smart contracts turn the terms of an agreement into computer code that executes when the contract terms are met.
 
-Smart contracts are the fundamental building blocks of [Ethereum applications](/dapps/). They are computer programs stored on the blockchain that allows us to convert traditional contracts into digital parallels. Smart contracts are very logical - following an if this then that structure. This means they behave exactly as programmed and cannot be changed.
+A smart contract is a computer program following "if X then Y" logic, which means they behave exactly as programmed and are resistant to participants' reneging on the terms of the contract. They are the fundamental building blocks of Ethereum's application layer.
```

Comment:
```suggestion
A smart contract is a computer program that often follows an "if this then that" logic, which means they behave exactly as programmed and are resistant to participants' reneging on the terms of the contract. They are the fundamental building blocks of Ethereum's application layer.
```

I think spelling out "if this then that" is more common sense & will make translations more straightforward.